components/automl/create_dataset_for_tables/component.py

```diff
@@ -0,0 +1,43 @@
+from typing import NamedTuple
+
+
+def automl_create_dataset_for_tables(
+    gcp_project_id: str,
+    gcp_region: str,
+    display_name: str,
+    description: str = '',
+    tables_dataset_metadata: dict = {},
+    retry=None, #=google.api_core.gapic_v1.method.DEFAULT,
+    timeout: float = None, #=google.api_core.gapic_v1.method.DEFAULT,
```

Comment:
Curious what is `google.api_core.gapic_v1.method.DEFAULT`?

Reply:
It's just a marker object that tells the client functions that the value was not passed. I wish they used `None` for that purpose.
https://github.com/googleapis/google-cloud-python/blob/902347776fc7b62aab3724a12dae87487f797408/api_core/google/api_core/gapic_v1/method.py#L27

Generally, Lightweight components support complex default values, but at this moment the functions still must be self-contained (unless code pickling is used). If the commented-out default was used, then function will stop being self-contained since it will require importing the `google` module.